docs/issue_template.md

```diff
@@ -1,14 +1,18 @@
-<!--- The problem --->
-1. Replace this text with a clear statement of the problem that needs to be solved including any relevant supporting files.
-- **If its a software abnormality**, explain how it should act, how it does act, and how to reproduce it.
-- **If you are reporting a security related problem,** DO NOT report here. Email [support@daemo.org](mailto:support@daemo.org) instead.
-- **If this is associated with a strategic milestone** add the `strategy` milestone.
-2. Add a title in the field above that concisely summarizes the problem.
-
-## My proposal
-3. If you have a solution to this problem and you will implement it, continue, otherwise remove the **my proposal** section.
+1. Add a proposal title above that concisely summarizes the problem or suggestion.
```

Comment:
I think adding the title after writing the problem (or proposal) leads to better titles. In the same way that writing a subject after writing an email leads to more informative subjects.

Reply:
Fine with me. I was mainly worried people would clear out all this text without getting that far down, and not read it.